frontend/src/App.jsx

```diff
@@ -4,6 +4,7 @@ import MainPage from "./components/MainPage/MainPage";
 import Result from "./components/ClericalReview/cl_e1/Result";
 import FinalResult from "./components/ClericalReview/cl_e3/FinalResult";
 import ClericalReviewMain from "./components/ClericalReview/cl_e2/ClericalReviewMain";
+import SelectSamplingAlgo from "./components/Research/Sampling/SelectSamplingAlgo"
```

Comment:
Add the symbol ";" at the end of the sentence.
```suggestion
import SelectSamplingAlgo from "./components/Research/Sampling/SelectSamplingAlgo";
```